src/app/modules/time-clock/services/entry.service.spec.ts

```diff
@@ -143,10 +143,13 @@ describe('EntryService', () => {
 
   it('entries are found by project id with a limit 2 by default', () => {
     const projectId = 'project-id';
+    const startDate = (moment().subtract(1, 'months')).format();
```

Comment:
I have a question.
What is the default format in this `format()` part?
I think you can add a specific format:  example .format('MM/DD/YYYY');  .format('MMM')

Reply:
The format used to send start_date and end_date in the path is 2021-03-10T15:42:13-05:00,  at the moment I think it should be sent like this